draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
@siyengar, over what period are your numbers taken?  I wouldn't mind losing 0.3% of 0-RTT given that we have a seamless fallback.  If this is over a day, that suggests we could have a narrower tolerance for drift than even 10s.  If this is just based on attempts at 0-RTT and is uncorrelated to the time between ticket issuance and use, then I'm not sure how to interpret the data in a way that would help us make a better decision.